modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -452,22 +176,22 @@ Before you add a new node to the cluster, check the management node can access t
 You also need to ensure that the node you want to add is registered to {productname}, and has a {caasp} channel assigned.
 
 
-.Procedure: Adding a Node to a Cluster
+.Procedure: Adding Nodes(s) to a Cluster
 . In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[Join Node].
-. Select the node to add from a list of available nodes.
+. Select the node(s) to add from a list of available nodes.
 The list of available nodes includes only nodes that are registered to {productname}, are not management nodes, and are not currently part of any cluster.
-. Follow the prompts to enter the {caasp} parameters for the node to be added.
-. OPTIONAL: Specify a custom ``ssh-agent`` socket that is valid only for the nodes that are being added.
-. Click btn:[Save] to schedule an action to add the node.
-During this action, {productname} prepares the node for joining by disabling swap, then joins the node to the cluster.
+. Follow the prompts to enter the {caasp} parameters for the node(s) to be added.
+. OPTIONAL: Specify a custom ``ssh-agent`` socket that is valid only for the node(s) that are being added.
+. Click btn:[Save] to schedule an action to add the node(s).
+During this action, {productname} prepares the node(s) for joining by disabling swap, then joins the node(s) to the cluster.
```

Comment:
Generally speaking, if you can use singular or plural, then we stick with plural. Adding the s in parentheses makes the doc harder to read and translate.

Reply:
Fixed